openapi/multi-yaml/components/schemas/PaymentError.yaml

```diff
@@ -0,0 +1,29 @@
+type: object
```

Comment:
Let's mark `network, network_error_category, network_error_code` as nullable, if they are nil we don't include them in the response